index.js

```diff
@@ -18,6 +18,8 @@ const flattenProperties = (props, sep, nestedChain = []) => {
             newProps = { ...newProps, $set: { ...props[key], ...flattenProperties(props[key], sep) } }
         } else if (key === '$set_once') {
             newProps = { ...newProps, $set_once: { ...props[key], ...flattenProperties(props[key], sep) } }
+        } else if (key === '$group_set') {
```

Comment:
Tiny refactoring tip: At this point we could just do `$set`, `$set_once` and `$group_set` in one branch with a `['$set', '$set_once', '$group_set'].includes(key)` condition, since their mechanics are exactly the same

Reply:
makes sense!

Reply:
ugh... but I really don't want to open a new editor for this repo again so I'll pass on doing this for now 🙈